src/iocore/net/UnixNetAccept.cc

```diff
@@ -478,10 +478,10 @@ NetAccept::acceptEvent(int event, void *ep)
     if ((res = accept_fn(this, e, false)) < 0) {
       Metrics::Gauge::decrement(net_rsb.accepts_currently_open);
       /* INKqa11179 */
-      Warning("Accept on port %d failed with error no %d", ats_ip_port_host_order(&server.addr), res);
+      Warning("Accept on port %d failed with error no %d", ats_ip_port_host_order(&server.accept_addr), res);
```

Comment:
Kinda strange, it looks like accept_addr it to specify the bind address, but addr is actually used internally.  Is this change required?

Reply:
This needs more investigation - I'm confused about why there are two variables and what the difference is.

Reply:
It looks like it uses accept_addr to specify what address/port to bind to, but then it reads the actual bound address into addr.  I'm not sure this needs both since it's not inheriting from Connection.

Reply:
These aren't interchangeable, because the address can be set to something different if `accept_addr` is not an IP:
```cpp
int
Server::listen(bool non_blocking, const NetProcessor::AcceptOptions &opt)
{
  ink_assert(!sock.is_ok());
  int       res = 0;
  socklen_t namelen;
  int       prot = IPPROTO_TCP;

  if (!ats_is_ip(&accept_addr)) {
    ats_ip4_set(&addr, INADDR_ANY, 0);
  } else {
    ats_ip_copy(&addr, &accept_addr);
  }
```

Reply:
I'm still not convinced we need both, but I think changing it to only one variable might require a behavior change.